olive/cli/quantize.py

```diff
@@ -75,6 +82,15 @@ def register_subcommand(parser: ArgumentParser):
         # dataset options
         add_dataset_options(sub_parser, required=False, include_train=False, include_eval=False)
 
+        quarot_group = sub_parser.add_argument_group(
```

Comment:
Devang is removing argument groups in https://github.com/microsoft/Olive/pull/1373. If we follow that here, I think it's better to name the arguments like `--quarot_rotate`, etc.